springdoc-openapi-starter-webmvc-api/src/test/resources/results/3.1.0/app8.json

```diff
@@ -0,0 +1,111 @@
+{
+  "openapi": "3.1.0",
+  "info": {
+    "title": "OpenAPI definition",
+    "version": "v0"
+  },
+  "servers": [
+    {
+      "url": "http://localhost",
+      "description": "Generated server url"
+    }
+  ],
+  "paths": {
+    "/": {
+      "get": {
+        "tags": [
+          "examples-controller"
+        ],
+        "operationId": "index",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "*/*": {
+                "schema": {
+                  "$ref": "#/components/schemas/ExamplesResponse"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "ExamplesResponse": {
```

Comment:
```suggestion
      "ExamplesResponse": {
        "type": "object",
```

Reply:
@JohnNiang It seems that the current unit test also not include `"type": "object"`

https://github.com/springdoc/springdoc-openapi/blob/d4f99ac9036a00ade542c3ebb1ce7780a171d7d8/springdoc-openapi-starter-webmvc-api/src/test/resources/results/3.1.0/app7.json#L37-L60

Reply:
See here: https://spec.openapis.org/oas/v3.1.0#data-types

> Data types in the OAS are based on the types supported by the [JSON Schema Specification Draft 2020-12](https://tools.ietf.org/html/draft-bhutton-json-schema-00#section-4.2.1).

In JSON Schema Specification Draft 2020-12, "type" field is optional, so this is valid JSON schema.

And the api-gen is done by swagger-core. We can advise swagger-core to provide `"type": "object"`.